git-workflows/skills/detect-conventional-commits/SKILL.md

```
@@ -0,0 +1,147 @@
+   - .commitlintrc.cjs
+   - .commitlintrc.ts
+   - commitlint.config.js
+   - commitlint.config.cjs
+   - commitlint.config.ts
+   ```
+
+2. Check glob results:
+   - IF any file found: Set has_commitlint = true
+   - IF no files found: Set has_commitlint = false
+
+**Early Return: Commitlint Found**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)